generators/app/templates/src/components/NavBar/children/Toolbar/toolbar.test.js

```diff
@@ -1,22 +1,21 @@
-import React from "react";
+/* eslint-disable jsx-a11y/label-has-associated-control */
```

Comment:
What is the reason for disabling these in this file?

Reply:
Fixed